Gemfile

```diff
@@ -19,6 +18,8 @@ gem 'nokogiri', '~> 1.6.2'
 gem 'pg'
 gem 'mixlib-authentication', '~> 1.3.0'
 gem 'sentry-raven'
+gem 'chef-web-core', git: 'https://github.com/chef/chef-web-core.git'
```

Comment:
you'll need to `bundle update chef-web-core` in order to pull in the changes that fix the roundedness of buttons.